src/components/Popover/_theme.scss

```diff
@@ -11,6 +11,8 @@ $color: var(--rui-Popover__color);
 $background-color: var(--rui-Popover__background-color);
 $box-shadow: var(--rui-Popover__box-shadow);
 
-$arrow-width: 1rem;
-$arrow-height: math.div($arrow-width, 2); // 1.
+$arrow-safe-rendering-overlap: 1px;
+$arrow-gap: 1px;
+$arrow-width: calc(1rem + #{$arrow-safe-rendering-overlap + $arrow-gap});
```

Comment:
On the other hand, only `$arrow-safe-rendering-overlap` needs to be here:

```suggestion
$arrow-width: calc(1rem + #{$arrow-safe-rendering-overlap});
```

Reply:
If i just do 
```css
$arrow-width: calc(1rem + #{$arrow-safe-rendering-overlap});
```
Then the arrow shrinks as I increase the arrow-gap
<img width="847" alt="Snímek obrazovky 2024-09-02 v 10 28 27" src="https://github.com/user-attachments/assets/cda9504b-31a3-42e9-8a94-93f810d43010">
<img width="903" alt="Snímek obrazovky 2024-09-02 v 10 28 19" src="https://github.com/user-attachments/assets/83511a6e-4ada-480f-b2dc-43055e99fac7">

What works is 
```css
$arrow-width: calc(1rem + #{$arrow-safe-rendering-overlap + ($arrow-gap * 2)});
```

Reply:
OK, let's try it this way.